### PR TITLE
Add MLflow UI messge to quickstart example and fix autolog spillage

### PR DESCRIPTION
**Pull request by stefannica**
## Describe changes
Add MLflow UI messge to quickstart example and fix autolog spillage

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: src/zenml/integrations/mlflow/experiment_trackers/mlflow_experiment_tracker.py [hunk 1]
```diff
@@ -229,6 +229,8 @@ def cleanup_step_run(
         """
         status = "FAILED" if step_failed else "FINISHED"
         mlflow_utils.stop_zenml_mlflow_runs(status)
+        breakpoint()
```
Comment (safoinme) [inline src/zenml/integrations/mlflow/experiment_trackers/mlflow_experiment_tracker.py:232]:
```suggestion
```

**Review by safoinme — CHANGES_REQUESTED** [review]

File: examples/quickstart/run.py [hunk 2]
```diff
@@ -75,6 +77,15 @@ def main():
     drift_detection_step = inf_run.get_step(step="drift_detector")
     EvidentlyVisualizer().visualize(drift_detection_step)
 
+    print(
+        "You can run:\n "
+        f"[italic green]    mlflow ui --backend-store-uri '{get_tracking_uri()}"
```
Comment (strickvl) [inline examples/quickstart/run.py:82]:
```suggestion
        f"[italic green]    mlflow ui --backend-store-uri '{get_tracking_uri()}'"
```

File: examples/quickstart_py37/run.py [hunk 3]
```diff
@@ -56,6 +58,15 @@ def main():
     drift_detection_step = inf_run.get_step(step="drift_detector")
     EvidentlyVisualizer().visualize(drift_detection_step)
 
+    print(
+        "You can run:\n "
+        f"[italic green]    mlflow ui --backend-store-uri '{get_tracking_uri()}"
```
Comment (strickvl) [inline examples/quickstart_py37/run.py:63]:
```suggestion
        f"[italic green]    mlflow ui --backend-store-uri '{get_tracking_uri()}'"
```

File: src/zenml/integrations/mlflow/experiment_trackers/mlflow_experiment_tracker.py (hunk 1 above)
Comment (strickvl) [inline src/zenml/integrations/mlflow/experiment_trackers/mlflow_experiment_tracker.py:232]:
```suggestion
```

**Review by strickvl — CHANGES_REQUESTED** [review]

File: src/zenml/integrations/mlflow/experiment_trackers/mlflow_experiment_tracker.py (hunk 1 above)
Comment (stefannica) [reply src/zenml/integrations/mlflow/experiment_trackers/mlflow_experiment_tracker.py:232]: hate makes waste

**Pull request merged**